addons/account_edi/views/account_move_views.xml

```diff
@@ -38,6 +42,10 @@
             <field name="arch" type="xml">
                 <xpath expr="//search/group/filter[@name='status']" position="after">
                     <filter string="Electronic invoicing state" name="edi_state" context="{'group_by': 'edi_state'}" />
+                    <filter name="to_check" position="after">
+                    <separator/>
+                    <filter string="Processing E-invoice" name="edi_to_process" domain="[('edi_state', 'in', [('to_send'), ('to_cancel')])]"/>
```

Comment:
Not sure about my naming here, but I can not find a better one immediately.

Reply:
To Check actually is already present so I removed it.
The name "E-invoicing process needed" is better for the other search.